website/pages/docs/plugins/sources/snyk/configuration.mdx

```diff
@@ -22,3 +22,9 @@ This is the (nested) spec used by the Snyk source plugin.
 
 - `endpoint_url` (string, optional. Default: not used):
   Endpoint URL to make the API requests to.
+
+- `retries` (integer, optional. Default: 10):
```

Comment:
```suggestion
- `retries` (`integer`, optional. Default: `10`):
```

Reply:
This would be different from the lines above it and other plugins afaict